Makefile

```diff
@@ -85,18 +85,18 @@ run-structurizr-export:
 	export -workspace /usr/local/structurizr/workspace.dsl -format mermaid
 
 scan-lpas: ##@app dumps all entries in the lpas dynamodb table
-	docker compose -f docker/docker-compose.yml exec localstack awslocal dynamodb scan --table-name lpas
+	docker compose -f docker/docker-compose.yml exec localstack awslocal dynamodb --region eu-west-1 scan --table-name lpas
```

Comment:
These commands weren't working for me after we bumped the localstack version. I played around with the shared DB env var but I was seeing some weirdness happening so this felt like a safer/less annoying option.

Reply:
ah yeah, I hadn't used these since but it is what I had to do to the localstack init